src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1407,4 +1414,34 @@ private void checkCreatingConsumer(String key, String value, HttpResponseStatus
             });
         consumer.get(TEST_TIMEOUT, TimeUnit.SECONDS);
     }
+
+    @Test
+    void createConsumerWithInvalidValueOfConfiguration(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        CompletableFuture<Boolean> create = new CompletableFuture<>();
+
+        JsonObject requestHeader = new JsonObject();
+        requestHeader.put("name", name);
+
+        LOGGER.info("Adding invalid value 'biary' to 'format' property configuration to invoke |422| status code");
+
+        requestHeader.put("format", "biary");
+        requestHeader.put("auto.offset.reset", "earliest");
+        requestHeader.put("fetch.min.bytes", "512");
+        requestHeader.put("consumer.request.timeout.ms", "30000");
+
+        consumerService()
+                .createConsumerRequest(groupId, requestHeader)
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(requestHeader, ar -> {
+                    context.verify(() -> {
+                        assertTrue(ar.succeeded());
+                        HttpResponse<JsonObject> response = ar.result();
```

Comment:
in all the test we expect an error we get an `HttpBridgeError` from the body testing the related error code and message other than the HTTP status code. Can you do the same here?